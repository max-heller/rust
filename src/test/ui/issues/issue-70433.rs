struct S;

fn main() {
    loop {
        let x;
        if true {
            x = S;
        }
        drop(x);
        //~^ ERROR use of possibly-uninitialized variable: `x` [E0381]
    }
}
