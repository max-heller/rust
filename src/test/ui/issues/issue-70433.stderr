error[E0381]: use of possibly-uninitialized variable: `x`
  --> $DIR/issue-70433.rs:9:14
   |
LL |         drop(x);
   |              ^ use of possibly-uninitialized `x`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0381`.
